modules/icgc-argo-workflows/prep/metrics/main.nf

```diff
@@ -8,8 +8,8 @@ process PREP_METRICS {
         'quay.io/biocontainers/python:3.8.3' }"
 
     input:
-    tuple val(meta), path(qc_files)
-    path multiqc
+    tuple val(meta), path(multiqc)
+    path qc_files // optional
 
     output:
     tuple val(meta), path('*.argo_metrics.json')   , emit: metrics_json
```

Comment:
Change output to reflect changes in:
https://github.com/icgc-argo-workflows/argo-modules/pull/39#discussion_r1666813639

Reply:
I updated the output file names in the 3 python scripts as compatible with the output definition here. eg `%s.argo_metrics.json` to `%s.dnaaln.argo_metrics.json` and `%s.metrics.json` to `%s.dnaaln.metrics.json`